ivy_tests/test_ivy/helpers/hypothesis_helpers/general_helpers.py

```diff
@@ -239,17 +233,18 @@ def get_mean_std(draw, *, dtype):
 
     Returns
     -------
-    A strategy that can be used in the @given hypothesis decorator.
+        A strategy that can be used in the @given hypothesis decorator.
     """
-    none_or_float = none_or_float = number_helpers.floats(dtype=dtype) | st.none()
+    # TODO: Does this actually work?
+    none_or_float = number_helpers.floats(dtype=dtype) | st.none()
```

Comment:
Yes, this is the same as writing `st.one_of(number_helpers.floats(dtype=dtype), st.none())`